openapi.yaml

```diff
@@ -2882,13 +2882,14 @@ paths:
     patch:
       summary: Modify a batch job
       operationId: update-job
-      description: >-
+      description: |-
         Modifies an existing job at the back-end, but maintains the identifier.
         Changes can be grouped in a single request.
 
+        The job status does not change.
 
         Jobs can only be modified when the job is not queued or running.
```

Comment:
And on a more general note: why can jobs only be changed when not queued/running? This is only about changing the metadata of a job, it should not interfere with the processing itself, right?

Reply:
You can also change the process graph and that is likely not a good idea while running. Similarly it's not a good idea to change the plan while runnnig. Also, the title etc are likely to get stored to the metadata, which may lead to strange behavior?

```suggestion
        Jobs can only be modified when the job is not queued and not running.
```

Reply:
> change the process graph ... Similarly, it's not a good idea to change the plan while running etc.

That would indeed be a bad idea for the user in the sense that they make things very confusing.  For the back-end it's probably less of a problem as it can use the process graph or plan at the time of starting the job.

Nevertheless, if this is a problem, then I think it's  also a bad idea to change these things even when the job is finished or failed.  So the constraint should be: jobs can only be modified when created or canceled